### ISSUE TITLE
ListStore does not implement GObjectTrait

### ISSUE THREAD

**Issue by drenninghoff** [gtk-rs/gtk#215]
When I try to get a ListStore object from a Glade file, Rust 1.5 complains that the GObjectTrait is not implemented. I believe the following code should be possible. 

Example code:
```rust
let liststore1: gtk::ListStore  = builder.get_object("liststore1").unwrap();
```

**Comment by gkoz** [comment]
This will will get fixed when the object reform (#59) lands, soon™.

**Comment by gkoz** [comment]
I believe this has been fixed by #221
